### DISCUSSION TITLE
Material data base help

### DISCUSSION THREAD

**Discussion by Steph152** [arkypita/LaserGRBL#1602] [Q&A]
Hello! I’m new to the world of laser engraving. I have an Atomstack A5. When I first started I didn’t understand how to use the material database. It was set to cut on acrylic but it engraved a wooden cutting board just fine. Now when I adjust the power and speed according to the material data base and set it for engrave and wood it doesn’t show up on the wood at all. Do I have something set up wrong?

**Comment by Tribo-2dot0** [comment]
It's strange, because engraving wood is very easy, even if it gets weak.
What if you increase power or decrease speed? Still not recording? If so, check the focus or maybe your laser (diode) has weakened for some reason.
